README.md

```diff
@@ -17,7 +17,7 @@ to its deployment configuration/health check rules).
 
 ### SSHing into a container
 
-`ecs-cluster ssh-service --cluster cluster-name --service service-name`
+`ecs-cluster ssh-service --cluster cluster-name --service service-name --user username`
```

Comment:
This user is related to foxpass user, right? Or does it work if you have a user on the box, and the box is setup for ssh access? I thought the `--user` part was related to Foxpass, but I don't see that mentioned anywhere in the readme, so I assume foxpass augments having a user account on the box.